packages/xo-web/src/common/intl/messages.js

```diff
@@ -1066,7 +1066,9 @@ const messages = {
   sshUserTooltip: 'Connect using external SSH tool as user…',
   sshUserLabel: 'SSH as…',
   sshUsernameLabel: 'SSH user name',
-  sshNeedClientTools: 'No IP address reported by client tools',
+  remoteNeedClientTools: 'No IP address reported by client tools',
+  rdp: 'RDP',
+  rdpRootTooltip: 'Connect using external RDP tool as root',
```

Comment:
I don't know RDP but if it's just a way to access the VM's desktop, then there's no such thing as root.

Reply:
That's correct. In theory, you could put credentials in the RDP URL, but there's no way to know those creds (in Windows, it depends on the language eg `Administrateur` or `Administrator`). So we'll use the simple RDP URL without any creds inside it.